static/monorail-2.0-sb.yaml

```diff
@@ -158,29 +158,73 @@ paths:
             $ref: '#/definitions/Error'
   /notification/progress:
     x-swagger-router-controller: notification
+# In some environment (such as ipxe), the avaiable http client doesn't support POST, so we have
+# to compromise to define below GET api to update progress information, but actually it does the
+# job of verb "POST"
     get:
       operationId: notificationProgressGet
```

Comment:
how about  using 'notificationProgressPost' directly,  it could avoid confusion in two places( another is "notificationProgressGet: notificationProgressPost" ) and avoid you have to comments  in two places.